css/main.css

```diff
@@ -272,6 +282,232 @@ nav {
   height: 850px;
 }
 
+/* Sponsors */
+.sponsors {
+  position: relative;
+  width: 100%;
+  margin: 0 auto;
+  color: #ffad0b;
+  font-family: "Avenir Light", sans-serif;
+}
+
+.sponsors .cropped {
+  position: relative;
+  width: 1440px;
```

Comment:
It actually won't mess up the full-site branch.  I added some css objects to continue the colored lines to the ends of the page if its larger than 1440px (this is so that the sponsorship image doesn't have to be resized).